website_odoo_debranding/static/src/js/disable_odoo.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * # -*- encoding: utf-8 -*-
+ * ##############################################################################
```

Comment:
Please use [shorter header](https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L3). Also the C/Python style `#` should be removed in favor of JS comment style.